### PR TITLE
Magnum Opus: Embolus

### PR DESCRIPTION
**Pull request by presheaf**
The only design choice I think is questionable is whether Embolus should prompt you if you have 0 creds. I elected to have it do so in part so I didn't have to think about whether there were any weird edge cases in which players might need the prompt in order to manually trigger card abilities to ensure they had the credit to add the counter. However, if anyone has strong opinions on this I'll happily change it, although I'll then want to add a test ensuring it works as well as one can expect with Launch Campaign.

### CODE REVIEW

File: src/clj/game/cards/upgrades.clj [hunk 1]
```diff
@@ -377,11 +377,30 @@
...
+          :effect (effect (end-run))}]
+     {:derezzed-events {:runner-turn-ends corp-rez-toast}
+      :events {:corp-turn-begins maybe-gain-counter
+               :successful-run {:req (req (pos? (get-counters card :power)))
+                                :msg "remove 1 power counter from Embolus"
+                                :effect (effect (add-counter card :power -1))}}
+      :abilities [maybe-gain-counter etr]})
+
    "Expo Grid"
-   (let [ability {:req (req (some #(and (is-type? % "Asset")
```
Comment (NoahTheDuke) [inline src/clj/game/cards/upgrades.clj]: Why this change?

Comment (presheaf) [reply src/clj/game/cards/upgrades.clj]: Oops, good question. It was certainly not intentional.
